supdup.c

```diff
@@ -524,7 +534,7 @@ sup_term (void)
  *    exit (1);
  *  }
  */
- 
```

Comment:
I think this whitespace cleanup is ok, but it should be in a separate commit.  Not mixed in with functional changes.

Reply:
Will do in another PR